charts/stardog/templates/configmap.yaml

```diff
@@ -14,7 +14,7 @@ data:
   {{- (.Files.Glob "files/log4j2.xml").AsConfig | nindent 2 }}
 
   {{- (.Files.Glob "files/stardog.properties").AsConfig | nindent 2 }}
-    pack.enabled=true
+    pack.enabled={{ if (eq (.Values.replicaCount | int) 1) }} false {{ else }} true {{ end }}
```

Comment:
Could you add quick check to the smoke tests to verify this? E.g. right after:

https://github.com/stardog-union/helm-charts/blob/master/tests/smoke.sh#L176

Create a single node deployment with this value set and then check that only a single Stardog pod starts and works and that no ZKs start?

Reply:
I would prefer that this be based on a flag and not `replicaCount == 1`.  It is valid to have a cluster of 1.  Further what would happen in this case if someone had a 3 node cluster but brought it down to 1 for a an interim maintenance period?

Reply:
Done - created a cluster.enabled flag and a test to verify the behaviour.